caktus/philly-hip
# [WIP] Update to community response notification signup form

### Description

This PR introduces updates to the `CommunityResponseSubscriber` model and related components.

**Field Changes:** 
- Added `organization_type`
- Added `mission_statement`
- Removed `organization_street_address`
- Removed  `organization_po_box`
- Removed `organization_zip_codes_served`
- Removed `title`

**Test Updates:**
- Removed outdated test cases related to removed fields
- No additional tests or test cases were added as they are not needed

**CSS Enhancements:**
- Added styling to make label asterisk appear red instead of black

---

### Related Tickets/Issues
https://app.clickup.com/t/868bevzhj

--- 

### Screenshots

Before:
<img width="494" alt="Screen Shot 2024-12-31 at 1 18 46 AM" src="https://github.com/user-attachments/assets/123763b7-03aa-4cca-b55c-d40c396022bc" />

After:
<img width="506" alt="Screen Shot 2024-12-31 at 1 19 55 AM" src="https://github.com/user-attachments/assets/3fcd769e-b474-49f5-ab3a-32e9f722d70c" />

---

### Testing Instructions

1. Run Migrations
```bash
python manage.py migrate
```
2. Verify functionality in the following areas:
- Form behavior for new/modified fields 
- Correct default values in the dropdown fields (see corresponding ticket for more information)

3. Run tests top ensure no regressions
```bash
make run-tests
```
